Jacob/html

ShaveYourEyebrows opened the pull request:
Add my HTML in pls

ShaveYourEyebrows committed:
Create HTML folder for frontend

ShaveYourEyebrows committed:
Adding initial HTML file and the favicon

A basic frontend with input fields for a student name, ID and course ID to enroll in

oOoOoOo0o commented on OBlock/html/frontend.html:
```diff
@@ -0,0 +1,17 @@
+	<head>
+		<meta charset='UTF-8'>
+		<meta name='viewport' content='width=device-width, initial-scale=1.0'>
+		<link rel="icon" href = "shrek.png" type= "image/png"/>
```
Let's not use Shrek

oOoOoOo0o commented on OBlock/html/shrek.png:
No Shrek

Bourkupine replied:
yeah lets not use shrek

Bourkupine commented on OBlock/html/frontend.html:
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang = 'en'>
```
lets remove the spacing around the '=' symbol for consistency

Bourkupine commented on OBlock/html/frontend.html:
```diff
@@ -0,0 +1,17 @@
+		<input type = "text" name = "studentname" placeholder="Enter the student name pls">
+		<input type = "int" name = "studentid" placeholder="Enter the student ID">
+		<input type = "int" name = "courseid" placeholder="Enter the course ID they want to enroll in">
+		<br></br>
```
dont think </br> is needed

Bourkupine replied:
forgot comments were md, meant the /br tag

Bourkupine commented on OBlock/html/init.txt:
remove this now

Bourkupine reviewed (changes requested):
Looks good, just minor nit picks

carthy27 reviewed (changes requested):
no shrek grrrr

ShaveYourEyebrows committed:
Delete OBlock/html/init.txt

Deleted unneeded init

ShaveYourEyebrows committed:
Improved frontend and JS

Made the frontend more robust and added a companion JS file for accepting inputs

ShaveYourEyebrows committed:
Removed spaces around =

ShaveYourEyebrows committed:
Removed Shrek

It's now a boring crypto image for CampusCash. Grr!

ShaveYourEyebrows committed:
Added an initial contract for a Student

Made a solidity contract to map a student object to a course code. This will be expanded with more functionality later in the week

ShaveYourEyebrows committed:
Deleted Shrek :'(
